--- src/client/refresh/gl3/header/local.h
+++ src/client/refresh/gl3/header/local.h
@@ -93,7 +93,8 @@
	GL3_ATTRIB_TEXCOORD   = 1, // for normal texture
	GL3_ATTRIB_LMTEXCOORD = 2, // for lightmap
	GL3_ATTRIB_COLOR      = 3, // per-vertex color
	GL3_ATTRIB_NORMAL     = 4, // vertex normal
-	GL3_ATTRIB_LIGHTFLAGS = 5  // uint, each set bit means "dyn light i affects this surface"
+	GL3_ATTRIB_LIGHTFLAGS = 5, // uint, each set bit means "dyn light i affects this surface"
+	GL3_ATTRIB_QCOORD     = 6  // for perspective-correct 2D trapezoids
 };

 // always using RGBA now, GLES3 on RPi4 doesn't work otherwise
